mne/io/base.py

```diff
@@ -801,6 +802,19 @@ def get_data(self, picks=None, start=0, stop=None,
             'bad' are omitted. If 'NaN', the bad samples are filled with NaNs.
         return_times : bool
             Whether to return times as well. Defaults to False.
+        units : str | dict | None
+            Specify the unit(s) that the data should be returned in. If None
+            (default), the data is returned in the channel type specific
+            default units. If str, must be a unit that will be used to scale
```

Comment:
Something like this probably
```suggestion
            default units, which are SI units (see :ref:`channel-units` and :term:`data channels`).
            If str, must be a unit that will be used to scale
```